.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           command: |
             git submodule init
             git submodule update
-      - run: DOCKER_BUILDKIT=1 BUILDKIT_PROGRESS=plain BUILD_FROM_INTERMEDIATE=y make build build_serialize
+      - run: DOCKER_BUILDKIT=1 BUILDKIT_PROGRESS=plain BUILD_FROM_INTERMEDIATE=y make build build_debug build_serialize
```

Comment:
So circleci will build the debug image but not run it for now? That seems like a good first thing to do.

Reply:
It does use it. Take a look at the `test_regression.py` below.

Reply:
huh, for some reason didn't see changes to that file before, got it